ark/api/v1alpha1/query_types.go

```diff
@@ -65,9 +65,13 @@ type QuerySpec struct {
 	Cancel bool `json:"cancel,omitempty"`
 }
 
+// Response defines a response from a query target.
```

Comment:
This description is great thanks, sorry to be super picky but it is `Content` that may be deprecated in the future but I don't think we need to document that for now as it's not decided